CMakeLists.txt

```diff
@@ -665,12 +690,13 @@ if(NOT WIN32 OR ROCKSDB_INSTALL_ON_WINDOWS)
 
   install(DIRECTORY include/rocksdb COMPONENT devel DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}")
 
+  set(INTERFACE_INCLUDE_DIRECTORIES, "${CMAKE_INSTALL_INCLUDEDIR}")
```

Comment:
cmake does not use `,` for separating function's parameters.

and also, `INTERFACE_INCLUDE_DIRECTORIES` is a property of cmake target. what is the "target" in this context?